api/metrics/src/main/java/io/opentelemetry/api/metrics/BoundDoubleCounter.java

```diff
@@ -5,20 +5,28 @@
 
 package io.opentelemetry.api.metrics;
 
+import io.opentelemetry.context.Context;
 import javax.annotation.concurrent.ThreadSafe;
 
-/** A {@code Bound Instrument} for a {@link DoubleCounter}. */
+/** A counter instrument that records {@code double} values with preassociated attributes. */
 @ThreadSafe
-public interface BoundDoubleCounter extends BoundSynchronousInstrument {
+public interface BoundDoubleCounter {
   /**
-   * Adds the given {@code increment} to the current value. The values cannot be negative.
+   * Reecord a value with prebound attributes.
```

Comment:
```suggestion
   * Records a value with prebound attributes.
```

For all the javadoc let's use present tense instead of command form

Reply:
Should be updated.